_blog/2018/soft-multitenancy.md

```diff
@@ -290,15 +276,35 @@ technology, ex. Kubernetes, rather than improvements in Istio capabilities.
 ## Future work
 Allowing a single Istio control plane to control multiple meshes would be an obvious next
 feature. An additional improvement is to provide a single mesh that can host different
-tenants with some level of isolation and security between the tenants.
-A [document](https://docs.google.com/document/d/14Hb07gSrfVt5KX9qNi7FzzGwB_6WBpAnDpPG6QEEd9Q)
+tenants with some level of isolation and security between the tenants.  This could be done
+by partitioning within a single control plane using the same logical notion of namespace as 
+Kubernetes. A [document](https://docs.google.com/document/d/14Hb07gSrfVt5KX9qNi7FzzGwB_6WBpAnDpPG6QEEd9Q)
 has been started within the Istio community to define additional use cases and the
 Istio functionality required to support those use cases.
 
-## Issues
+## Issues, limitiations or caveats
+### Known issues 
 * The CA (Certificate Authority) and mixer Istio pod logs from one tenant's Istio control
```

Comment:
Again in view mode, not sure we need the bullet now (asterisk).
Once I merge your change in, I'm going to read over again and probably make some additional changes.